### PR TITLE
Parallel integration tests

### PR DESCRIPTION
**Pull request by ly-kolena**
### Please check if the PR fulfills these requirements

- [ ] Include reference to internal ticket and/or GitHub issue "Fixes #NNNN" (if applicable)
- [ ] Relevant tests for the changes have been added
- [ ] Relevant docs have been added / updated

### CODE REVIEW

**Comment by gordonhart** [comment]
> 12 minutes

👀

**Review by munkyshi — APPROVED** [review]
Oh heck yes, this looks great

File: .circleci/config.yml [hunk 1]
```diff
@@ -163,11 +208,18 @@ workflows:
...
+            parameters:
+              python-version: [ "3.9" ]
+              pytest-group: [ detection, fr, generic ]
+          requires:
+            - ci-base-<< matrix.python-version >>
       - integration-test:
           matrix:
             parameters:
               python-version: [ "3.9" ]
-              pytest-group: [ detection, classification, fr, generic, misc ]
```
Comment (gordonhart) [inline .circleci/config.yml:170]: Any reason not to parallelize `classification` and `misc` too, removing the need for separate `integration-test` and `integration-test-parallel` job definitions?

Comment (ly-kolena) [reply .circleci/config.yml:170]: could revisit and see.

**Pull request merged**